cmd/hlb/command/run.go

```diff
@@ -205,7 +206,14 @@ func Run(ctx context.Context, cln *client.Client, rc io.ReadCloser, opts RunOpti
 		return solveReq.Solve(ctx, cln, p.MultiWriter())
 	})
 
-	return p.Wait()
+	err = p.Wait()
+	if err != nil {
+		for _, source := range errdefs.Sources(err) {
+			source.Print(os.Stderr)
```

Comment:
We should probably add an `ErrOutput os.Writer` to the RunOptions so we can use that here and plumb it through from newt.